pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1506,6 +1506,8 @@ func (dm *DockerManager) killContainer(containerID kubecontainer.ContainerID, co
 		select {
 		case <-time.After(time.Duration(gracePeriod) * time.Second):
 			glog.V(2).Infof("preStop hook for container %q did not complete in %d seconds", name, gracePeriod)
+			message := fmt.Sprintf("preStop hook for container %q did not complete in %d seconds", name, gracePeriod)
```

Comment:
What does it mean for a hook to complete? Is it the common terminology? What about "did not finish"?